.github/workflows/integration_tests.yml

```diff
@@ -1118,20 +1125,31 @@ jobs:
           - SERVICE_NAME: helm-service
     env:
       SERVICE_NAME: ${{ matrix.SERVICE_NAME }}
-      META_KEPTN_KEPTN_PROJECT: keptn
+      KEPTN_ARCHIVE_PATH: ~/downloads/keptn-archive.tar.gz
     steps:
+      - name: Create downloads folder
+        run: mkdir ~/downloads
+
       - name: Cache Keptn CLI
         id: keptn-cli-cache
         uses: actions/cache@v2
         with:
-          path: /usr/local/bin/keptn
+          path: ~/downloads
           key: ${{ env.META_KEPTN_VERSION }}
 
-      - name: Install Keptn CLI for META_KEPTN
-        id: install_keptn_cli
+      - name: Download Keptn CLI for META_KEPTN
+        id: download_keptn_cli
         if: steps.keptn-cli-cache.outputs.cache-hit != 'true'
         run: |
-          curl -sL https://get.keptn.sh | KEPTN_VERSION=${{ env.META_KEPTN_VERSION }} bash
+          curl -sL https://github.com/keptn/keptn/releases/download/${META_KEPTN_VERSION}/keptn-${META_KEPTN_VERSION}-linux-amd64.tar.gz -o ${{ env.KEPTN_ARCHIVE_PATH }}
+          tar -xvf ${{ env.KEPTN_ARCHIVE_PATH }}
+          mv ./keptn-${META_KEPTN_VERSION}-linux-amd64 keptn
+          chmod +x keptn
+          mv ./keptn /usr/local/bin/keptn
```

Comment:
This might require `sudo`

Reply:
i will try it out and add it if there's no other way

Reply:
it seems that no `sudo` is needed :)
https://github.com/keptn/keptn/runs/2894630082?check_suite_focus=true#step:5:4